packages/config/src/lib/createPlugins.ts

```diff
@@ -2,7 +2,7 @@ import { LogType, fecLogger } from '@redhat-cloud-services/frontend-components-c
 import { Compiler, WebpackPluginInstance } from 'webpack';
 import { CommonConfigOptions } from './createConfig';
 
-const { SourceMapDevToolPlugin } = require('webpack');
+import { SourceMapDevToolPlugin } from 'webpack';
```

Comment:
The constructor below was still raising an error when starting webpack. Changing this import to just `import webpack form 'webpack'` and the constructor below to `new webpack.SourceMapDevToolPlugin(` seemed to solve the issues.

Reply:
OK let me change it.